.github/workflows/build_images.yml

```diff
@@ -3,7 +3,8 @@ name: Build Docker images for all OpenSlides services
 on: [push, workflow_dispatch]
 
 env:
-  IMAGE_VERSION: 4.0.0-dev
+  IMAGE_VERSION: 4.0-dev
```

Comment:
Please change this back to 4.0.0-dev to get a correct semver version (https://semver.org)